ocs_ci/ocs/resources/csv.py

```diff
@@ -0,0 +1,71 @@
+"""
+CSV related functionalities
+"""
+import logging
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+class CSV(OCP):
+    """
+    This class represent ClusterServiceVersion (CSV) and contains all related
+    methods we need to do with CSV.
+    """
+
+    def __init__(self, name="", *args, **kwargs):
+        """
+        Initializer function for CSV class
+
+        Args:
+            name (str): Name of CSV
+
+        """
+        super(CSV, self).__init__(*args, **kwargs)
+        self.name = name
+
+    def check_phase(self, phase):
+        """
+        Check phase of CSV resource
+
+        Args:
+            phase (str): Phase of CSV object
+
+        Returns:
+            bool: True if phase of object is the same as passed one, False
+                otherwise.
+
+        """
+        data = {}
+        try:
+            data = self.get(resource_name=self.name)
+        except CommandFailed:
+            logger.info(f"Cannot find CSV object {self.name}")
```

Comment:
I think you can return False right after this line instead of trying to check data['status']['phase'] == phase

Reply:
done